shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -23,8 +23,8 @@
             <description>When enabled, the motors will spin at the ChannelNeutral command when armed with zero throttle.</description>
         </field>
 
-        <field name="MotorInputOutputCurveFit" units="-" type="float" elementnames="A,B" defaultvalue="1,1">
-            <description>Actuator mapping of input in [-1,1] to output on [-1,1], using power equation of type a*x^b.</description>
+        <field name="MotorInputOutputCurveFit" units="-" type="float" elements="1" defaultvalue="1">
```

Comment:
(You still haven't been able to explain what this is for-- is it deliberate for the clipping behavior?  Are you using this to avoid the upper end of the actuator range where there's saturation effects?)   Without some kind of justification of what this is trying to do, it doesn't belong here.  And if you're using it to deliberately clip, rather than as a pure multiplication operation, that needs to be documented and justified.